client/src/ui/Form.jsx

```diff
@@ -326,6 +351,103 @@ export default class Form extends Component {
     }
   }
 
+  getGradeOptions() {
+      const gradeOptions = [];
+      this.gradeRecieved = ["A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", "F"];
+      this.gradeRecieved.forEach((grade) =>
+        gradeOptions.push({
+          "value": grade,
+          "label": grade
+        }))
+      
+      return gradeOptions
+    }
+
+  getMajorOptions() {
+      const majorOptions = [];
+      this.majorsOptions = ["Computer science",
```

Comment:
where did we get this list form? It seems pretty inaccurate. For example, ECE is "Electrical and electronics engineering," ILR is "Labor and industrial relations," and (what I assume is) FGSS is somehow listed as "Gay/lesbian studies."

could we also look into using an endpoint for this—I know we have a list of majors somewhere for our search?

Reply:
Also either way we should have this list in another file so that this component isn't super long.